Formula/osc-cli.rb

```diff
@@ -0,0 +1,38 @@
+class OscCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Official Outscale CLI providing connectors to Outscale API"
+  homepage "https://github.com/outscale/osc-cli"
+  url "https://files.pythonhosted.org/packages/8c/29/cf68365132a0ac906278abdc704569ce8f1ba4c609a2067cd00c5ed22271/osc-sdk-1.6.tar.gz"
+  sha256 "fb312c82c73ed4eba7bbb3a028e4d2f53e00e28c576ff2f638258f27a855e7e7"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.9"
+
+  resource "fire" do
+    url "https://files.pythonhosted.org/packages/5a/b7/205702f348aab198baecd1d8344a90748cb68f53bdcd1cc30cbc08e47d3e/fire-0.1.3.tar.gz"
+    sha256 "c299d16064ff81cbb649b65988300d4a28b71ecfb789d1fb74d99ea98ae4d2eb"
+  end
+
+  resource "request" do
+    url "https://files.pythonhosted.org/packages/52/2c/514e4ac25da2b08ca5a464c50463682126385c4272c18193876e91f4bc38/requests-2.21.0.tar.gz"
+    sha256 "502a824f31acdacb3a35b6690b5fbf0bc41d63a24a45c4004352b0242707598e"
+  end
+
+  resource "xmltodict" do
+    url "https://files.pythonhosted.org/packages/57/17/a6acddc5f5993ea6eaf792b2e6c3be55e3e11f3b85206c818572585f61e1/xmltodict-0.11.0.tar.gz"
+    sha256 "8f8d7d40aa28d83f4109a7e8aa86e67a4df202d9538be40c0cb1d70da527b0df"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # we test the help wich is printed in stderr :(
+    str = shell_output("#{bin}/osc-cli -- --help 2>&1 >/dev/null")
+    assert_includes str, "SERVICE"
+    str = shell_output("#{bin}/osc-cli api ReadVms 2>&1 >/dev/null", 1)
+    assert_includes str, "No configuration file found in home folder"
```

Comment:
Are you sure you want to redirect to `/dev/null` here? That doesn't sound right. In any case:
```suggestion
    assert_match "SERVICE", shell_output("#{bin}/osc-cli -- --help 2>&1 >/dev/null")
    assert_match "No configuration file found in home folder", shell_output("#{bin}/osc-cli api ReadVms 2>&1 >/dev/null", 1)
```

Reply:
`assert_match` matches only substrings. For example, the first `assert_match` test will pass as long as the `shell_output` call includes the substring `SERVICE`, which it presumably does based on your initial test.